src/app/components/client/exposure_card/ExposureCardDataClass.tsx

```diff
@@ -122,7 +121,6 @@ export const ExposureCardDataClassLayout = (
   return (
     <div className={styles.detailsFoundItem}>
       <div className={styles.label}>
-        <span className={styles.exposureTypeIcon}>{props.icon}</span>
```

Comment:
We can also remove the style for `.exposureTypeIcon` [here](https://github.com/mozilla/blurts-server/blob/mntor-3378/src/app/components/client/exposure_card/ExposureCardDataClass.module.scss#L10).